### PR TITLE
update README_zh.md, syncing from README.md

### PR DESCRIPTION
**Pull request by cl2017**
Signed-off-by: chenlin.liu <chenlin.liu@daocloud.io>

**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
update README_zh.md, syncing from README.md

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: README_zh.md [hunk 1]
```diff
@@ -11,30 +11,35 @@
...
+- **云边可靠协作**：在不稳定的云边网络上，可以保证消息传递的可靠性，不会丢失。
 
-### 简化开发
+- **边缘自治**：当云边之间的网络不稳定或者边缘端离线或重启时，确保边缘节点可以自主运行，同时确保边缘端的应用正常运行。
 
-开发人员可以编写常规的基于 http 或 mqtt 的应用程序，容器化并在边缘或云端任何地方运行。
+- **边缘设备管理**：通过 Kubernetes 的原生API，并由CRD来管理边缘设备。
 
-### Kubernetes 原生支持
+- **非常轻量级的边缘代理**：在资源有限的边缘端上运行的非常轻量级的边缘代理(EdgeCore)。
```
Comment (fisherxu) [inline README_zh.md:33]:
```suggestion
- **极致轻量的边缘代理**：在资源有限的边缘端上运行的非常轻量级的边缘代理(EdgeCore)。
```

**Review by fisherxu — COMMENTED** [review]
Thanks for the fixing!

**Comment by zc2638** [comment]
Please squash  to one commit.

**Comment by cl2017** [comment]
>

**Comment by cl2017** [comment]
> 

ok, done

**Review by fisherxu — COMMENTED** [review]
/approve

**Pull request merged**